File: Python/컴퓨터공학개론/chap.6/1.py
#1번 문제, C135333, 임원재

list_num = []
sum = 0

while True:
    num = int(input("정수를 입력하시오: "))
    if num < 0:
        break
    list_num.append(num)
    sum += num
    
avg = sum / len(list_num)

print("평균= %0.1f" %avg)

cnt = 0
for i in range(len(list_num)):
    if list_num[i] > avg:
        cnt += 1

print("평균을 상회하는 숫자의 개수= %d" %cnt)

File: C/study/2주차 과제/1.c
#include <stdio.h>

int main(){
    int n;
    int sum = 0;
    printf("입력할 정수의 개수를 입력하시오 : ");
    scanf("%d\n", &n);

    int input;
    for (int i = 0; i < n; i++){
        scanf("%d", &input);
        sum += input;
    }

    double avg = sum / n;
    printf("평균 : %lf\n", avg);
    
    return 0;
}

